Update readme.md (#690)
mattermost/mattermost-plugin-jira: readme.md

## Development

> To contribute to the project see https://www.mattermost.org/contribute-to-mattermost.

Reviewer: That page seems quite generic. Someone who is looking into contributing to a plugin might not be interested in other parts of Mattermost. Maybe move this at the end of the section and make clear that it's about other Mattermost projects?

This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.

## Support

- For Mattermost customers - Please open a support case.

> - For Questions, Suggestions, and Help - Please find us on our forum at https://forum.mattermost.org/c/plugins.

Author: ```suggestion
- For Questions and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.
```

> - For Questions and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.

Reviewer 2: ```suggestion
- For questions and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.
```

> - For suggestions and feedback, you can [open an issue](https://github.com/mattermost/mattermost-plugin-jira/issues) or post in the Jira Plugin channel.

Reviewer: Should we explicitly point people who report bugs to the issues and everyone else to the channel? The line between question and feedback can be thin.

Reviewer 2: I kind of agree with @hanzei.  We should direct questions, suggestions, and help, to visit the mattermost community server jira plugin channel. 
To report a bug, use github issues.  

I'm pre-approving because I am out this week. Thanks for the PR and fixes, @justinegeffen

Author: ```suggestion
- To report a bug, please [open an issue](https://github.com/mattermost/mattermost-plugin-jira/issues).
```

## Development

This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.

For more information about contributing to Mattermost, and the different ways you can contribute, see https://www.mattermost.org/contribute-to-mattermost.

## Support

- For Mattermost customers - Please open a support case.
- For questions, suggestions, and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.
- To report a bug, please [open an issue](https://github.com/mattermost/mattermost-plugin-jira/issues).

> Please read the Jira plugin's [developer docs](https://mattermost.gitbook.io/plugin-jira/development/environment) to learn more about contributing to the project.

Reviewer: That doesn't seem right. Shouldn't that text replace the first part in `Development`?

## Development

This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.

For more information about contributing to Mattermost, and the different ways you can contribute, see https://www.mattermost.org/contribute-to-mattermost.

## Support

- For Mattermost customers - Please open a support case.

> - For questions and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.

Author: ```suggestion
- For questions, suggestions, and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.
```

This repository is licensed under the Apache 2.0 License, except for the [server/enterprise](server/enterprise) directory which is licensed under the [Mattermost Source Available License](LICENSE.enterprise). See [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.

## Development

To contribute to the project see https://www.mattermost.org/contribute-to-mattermost.

This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.

## Support

- For Mattermost customers - Please open a support case.

> - For Questions, Suggestions, and Help - Please find us on our forum at https://forum.mattermost.org/c/plugins.

Reviewer: Should we point people towards https://github.com/mattermost/mattermost-plugin-jira/issues and the Jira plugin channel instead? The integrations team barely uses the forum.

Reviewer 2: The idea of this ticket is to add all people signed in into MSTeams Calling that are on the channel to the meeting.

Reviewer: @justinegeffen What do you think about this?

Author: I missed this comment but I agree and have updated the text accordingly. :)
